### PR TITLE
fix: licensing policy and arn info

### PR DESCRIPTION
**Pull request by vacas5**
Dimitri found when testing that I had forgotten to add the policy
information for consuming a license. Also, it was difficult to follow
where to get the license ARN from AWS.

Adding a prerequisite to the add licensing page because this is where
users are directed from the download page. The purchase page I felt was
superfluous and not linked anywhere, so I melded that into the existing
index.

Also, added a bit more color about how to find the license ARN. We want
to make sure we can test this end to end.

Closes D2IQ-82950
Closes D2IQ-82951

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [x] Create new PRs against `develop`, or `main` for hotfixes to production.
- [x] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.1/licensing/add/index.md [hunk 1]
```diff
@@ -7,9 +7,21 @@ menuWeight: 5
 beta: false
 ---
 
-## Enter a Valid License Key
+## Prerequisites
 
-After you download the license, an administrator must add it to Kommander.
+For licenses purchased via the [AWS Marketplace](https://aws.amazon.com/marketplace/), an AWS administrator must attach the [AWS managed policy](https://docs.aws.amazon.com/license-manager/latest/userguide/security-iam-awsmanpol.html#security-iam-AWSLicenseManagerConsumptionPolicy) `AWSLicenseManagerConsumptionPolicy` to the `AWSIAMRoleControlPlane` role created when [configuring AWS IAM policies for Konvoy](/dkp/konvoy/latest/choose-infrastructure/aws/iam-policies/).
```
Comment (dkoshkin) [inline pages/dkp/kommander/2.1/licensing/add/index.md:12]:
```suggestion
For licenses purchased via the [AWS Marketplace](https://aws.amazon.com/marketplace/), an AWS administrator must attach the [AWS managed policy](https://docs.aws.amazon.com/license-manager/latest/userguide/security-iam-awsmanpol.html#security-iam-AWSLicenseManagerConsumptionPolicy) `AWSLicenseManagerConsumptionPolicy` to the `control-plane.cluster-api-provider-aws.sigs.k8s.io` role created when [configuring AWS IAM policies for Konvoy](/dkp/konvoy/latest/choose-infrastructure/aws/iam-policies/).
```

**Review by samvantran — COMMENTED** [review]
👍  looks good with dimitri's change

File: pages/dkp/kommander/2.1/licensing/add/index.md [hunk 2]
```diff
@@ -7,9 +7,21 @@ menuWeight: 5
...
+
+## Obtain a license token or AWS license ARN
+
+For licenses purchased directly from D2iQ, obtain the license token via the customer support portal. Input this token in the last step below.
+
+For licenses purchased via the AWS marketplace, find the license in the "Granted Licenses" table of AWS License Manager inside the AWS console. If necessary, modify the table view preferences to show "License ARN." Copy this value for use in the last step below.
+
+## Enter license information
+
+An administrator must add licenses to Kommander.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/licensing/add/index.md:24]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>An administrator must add licenses to Kommander.</p>
```

File: pages/dkp/kommander/2.1/licensing/add/index.md [hunk 3]
```diff
@@ -7,9 +7,21 @@ menuWeight: 5
...
 
 In the Kommander UI, do the following:
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/licensing/add/index.md:26]:
```suggestion
From the Kommander UI, complete the following steps:
```

File: pages/dkp/kommander/2.1/licensing/index.md [hunk 4]
```diff
@@ -7,4 +7,13 @@ menuWeight: 299
...
-The following instructions describe how to download and license Kommander.
\ No newline at end of file
+The following instructions describe how to download and license Kommander.
+
+## Purchase a License
+
+To obtain a valid Kommander license:
+
+- Contact your sales representative at <sales@d2iq.com> to purchase one or more licenses.
+- Purchase a license via the [AWS Marketplace](https://aws.amazon.com/marketplace/). After purchase, your license information (ARN) is accessible in the AWS License Manager console.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/licensing/index.md:17]:
```suggestion

- Purchase a license via the [AWS Marketplace](https://aws.amazon.com/marketplace/). After purchase, your license information (ARN) is accessible in the AWS License Manager console.
```

Comment (Colin1841) [inline pages/dkp/kommander/2.1/licensing/index.md:17]: I think these are okay with out a space between the bullets, correct?

**Review by Colin1841 — APPROVED** [review]

**Review by dkoshkin — APPROVED** [review]

**Pull request merged**